tests/pester-tests/commands/choco-upgrade.Tests.ps1

```diff
@@ -695,6 +695,44 @@ To upgrade a local, or remote file, you may use:
         }
     }
 
+    Context 'Upgrading a dependency with pinned parent and depends on a range less than upgrade version' {

```

Comment:
I think this test is fine more or less, but it needs to include comments detailing why this behaviour is expected, referencing the dependency tree of the packages we're working with, so it's clear why we're expecting this behaviour exactly. Without going and looking at all the packages it uses it's a little opaque.

Reply:
Agreed. I have added a comment pointing at the integration test I took this test from. I've also added a summary to that test, that should hopefully describe what is expected in it.